app/views/shared/_footer.html.erb

```diff
@@ -1 +1,53 @@
-<%= "Argo v#{Argo.version}" %>
```

Comment:
We don't really bump this version, so I ditched it instead of trying to find a way to fit it into this updated design. 🤷🏻‍♂️ Does anyone use it or find it meaningful? @andrewjbtw @sul-dlss/infrastructure-team

Reply:
That's not an ancient symbol etched into Argo? I'm not sure it's changed in the time I've been at SUL.

Reply:
So, no, I don't find it useful or meaningful (in the way it's intended, anyway).

Reply:
> That's not an ancient symbol etched into Argo? I'm not sure it's changed in the time I've been at SUL.

a long long time ago, we were in the habit of versioning argo.  i was mildly in favor of keeping that up at one point, but we were never super consistent about it, and we haven't bumped the version since 2019 (i'm actually slightly surprised the last bump was that recent).  i'd now be in favor of just removing the [`VERSION` file](https://github.com/sul-dlss/argo/commits/main/VERSION) and related uses of it.  maybe as a follow on tech debt ticket?  i wouldn't tie this PR to that bit of cleanup work, but it seems to me like something we can consider defunct now.  i'm happy to file the ticket if that sounds agreeable.

Reply:
i went ahead and filed the ticket, we can close it if it turns out we don't want to do away with argo versioning after all: https://github.com/sul-dlss/argo/issues/4152

Reply:
Thanks, @jmartin-sul & @andrewjbtw

Reply:
Mike, if you remove the VERSION file, is this is the only reference to it?  If so, you can close #4152 at the same time.   This is definitely vestigial and now pointless.

Reply:
@ndushay I'm not planning to include #4152 work in this PR. A fun, small ticket for someone else when they get bored. 😉